art/configs/logger/neptune.yaml

```diff
@@ -1,5 +1,5 @@
 _target_: lightning.pytorch.loggers.NeptuneLogger
 api_key: ${oc.env:NEPTUNE_API_TOKEN} # api key is loaded from environment variable
-project: skdbmk/wake-word-detection
+project: skdbmk/sourceseparation
```

Comment:
maybe we can have one big Neptune project, where we can train several tasks at once? It'd just require adding a label to each run

Reply:
to be discussed. For me many projects are better.